models/t_caa.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+// SetTargetCAA sets the CAA fields.
+func (rc *RecordConfig) SetTargetCAA(flag uint8, tag string, target string) error {
```

Comment:
Would `SetCAATarget` be a more appropriate name? This applies to almost all methods with that pattern.

Reply:
It is more searchable to have SetTarget*